client/landing/stepper/declarative-flow/internals/steps-repository/site-vertical/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { StepContainer } from '@automattic/onboarding';
+import { useTranslate } from 'i18n-calypso';
+import React from 'react';
+import siteVerticalImage from 'calypso/assets/images/onboarding/site-vertical.svg';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { useQuery } from '../../../../hooks/use-query';
+import SiteVerticalForm from './form';
+import type { Step } from '../../types';
+
+const SiteVertical: Step = function SiteVertical( { navigation } ) {
+	const { goNext, submit } = navigation;
+	const translate = useTranslate();
+	const headerText = translate( 'What’s your website about?' );
+	const subHeaderText = translate( 'Choose a category that defines your website the best.' );
+	const isSkipSynonyms = useQuery().get( 'isSkipSynonyms' );
+
+	const handleSubmit = () => {
+		submit?.();
```

Comment:
Have we figured out how to store the selected vertical?

Reply:
I'll create a PR for this! I'd like to maintain the scope for this PR.

Reply:
Ah, sure!